src/StackExchange.Redis/APITypes/BitfieldCommandBuilder.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System.Collections.Generic;
+
+namespace StackExchange.Redis;
+
+/// <summary>
+/// Builder for bitfield commands that take multiple sub-commands.
+/// </summary>
+public class BitfieldCommandBuilder
+{
+    private readonly LinkedList<RedisValue> _args = new LinkedList<RedisValue>();
```

Comment:
Been a little while since I put this together so my recollection might be a bit off - IIRC my thinking was:

1. We don't know the size of the array ahead of time - hence we can't really initialize a List<T> without knowing it won't need to be resized.
2. LinkedLists allow O(1) tail insertion
3. We only need to enumerate when sending it, hence the O(N) scan is what you'd expect anyway.

Reply:
Though now that I think of it, I suppose you are performing allocation each time you perform the addlast 🤔

Reply:
frankly, linked-list just barely gets used - I'd sooner use a `List<T>`, but again I wonder whether this is actually a `List<SomeUnionStructThatIsGetSetAndIncrby>`, so each element in the list is not an *argument* but a *logical operation* - this might also make it much easier to do very efficient single-shot operations, which I *expect* to be relatively common. Let me have a think here - I like the *ideas* in this PR, but I think we can iterate the API a little.